cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -348,7 +348,7 @@ spec:
 
         # The logging level for the operator: INFO | DEBUG
         - name: ROOK_LOG_LEVEL
-          value: "INFO"
+          value: "DEBUG"
```

Comment:
I'd suggest you keep a separate commit for temporary testing changes so it's easier to revert later.